gazelle/python/configure.go

```diff
@@ -181,6 +182,13 @@ func (py *Configurer) Configure(c *config.Config, rel string, f *rule.File) {
 			}
 		case pythonconfig.Visibility:
 			config.AppendVisibility(strings.TrimSpace(d.Value))
+		case pythonconfig.TestFilePattern:
+			value := strings.TrimSpace(d.Value)
+			if value == "" {
+				log.Fatal("ERROR: Directive 'python_test_file_pattern' requires a value.")
```

Comment:
nit: there is no need to double severity here as `Fatal` will be already in the log. also, go's error messages are conventionally lowercased since they can be nested (e.g. `something bad happened: directive 'python...`)
```suggestion
				log.Fatal("directive 'python_test_file_pattern' requires a value.")
```

Reply:
TIL, thanks! Updated.

As an FYI, the `ERROR:` and `\n` were included to maintain consistency with the other `log.Fatal` messages:

```shell
$ rg 'log.Fatal.?\("'
manifest/generate/generate.go
90:             log.Fatalf("ERROR: %v\n", err)
109:            log.Fatalf("ERROR: %v\n", err)

pythonconfig/pythonconfig.go
444:                    log.Fatalf("ERROR: Failed to compile glob '%v'. Error: syntax error in pattern\n", p)

python/lifecycle.go
41:                     log.Fatalf("failed to write parser zip: %v", err)
47:                     log.Fatalf("cannot write %q: %v", helperPath, err)

python/generate.go
238:                    log.Fatalf("ERROR: %v\n", err)
319:                    log.Fatalf("ERROR: %v\n", err)
353:                    log.Fatalf("ERROR: %v\n", err)
384:                    log.Fatalf("ERROR: %v\n", err)
504:            log.Fatalf("ERROR: %v\n", err)

python/configure.go
188:                            log.Fatal("ERROR: Directive 'python_test_file_pattern' requires a value.")
```

Prior to this PR, 7 of 9 calls included both `ERROR:` and `\n`.